opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -2601,6 +2605,8 @@ namespace Opm {
                     perfPhaseRate = perf_phase_rate[ perf*np + phaseIdx ];
                     weight_factor += cellDensity  * perfPhaseRate/cellBinv * cellInternalEnergy/cellTemperatures;
                 }
+                //NB hack better method in progress
+                weight_factor = std::abs(weightfactor)+1e-13;
```

Comment:
What is the purpose of this hack?

Reply:
To avoid zero incase of perfect crossflow.

Reply:
weightfactor --> weight_factor?

Reply:
hopefully fixed (forgot a push)